app/constants/url.js

```diff
@@ -1 +1,9 @@
 export const GITHUB_URL = 'https://github.com/Real-Dev-Squad/website-my';
+export const RDS_MAIN_URL = 'http://realdevsquad.com/';
+export const RDS_WELCOME_URL = 'https://welcome.realdevsquad.com/';
+export const RDS_EVENTS_URL = 'http://realdevsquad.com/events.html';
+export const RDS_MEMBERS_URL = 'https://members.realdevsquad.com/';
+export const RDS_CRYPTO_URL = 'https://crypto.realdevsquad.com/';
+export const RDS_STATUS_URL = 'https://status.realdevsquad.com/';
+export const LOGOUT_URL = 'https://api.realdevsquad.com/auth/signout';
```

Comment:
API Base Url can be imported from `config/environment.js` and then I think we can use it directly in `controllers/application.js`

Reply:
![Uploading image.png…]()

Reply:
Resolved.